src/main/java/org/opensearch/security/dlic/rest/api/FlushCacheApiAction.java

```diff
@@ -43,16 +43,18 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import org.opensearch.security.securityconf.impl.CType;
-
```

Comment:
Please undo all white space changes (it introduces unnecessary diffs that may cause merge conflicts later).